src/js/components/Graphs/Timeline/timeline.jsx

```diff
@@ -16,29 +16,27 @@
 
 import React, { useEffect } from 'react';
 import PropTypes from 'prop-types';
-import { constructTimeline } from '../../helpers/timeline/declarationHelper';
-import { drawTimeline } from '../../helpers/timeline/drawHelper';
-import { TimelineContainer, Timeline } from './timeline.style';
-
-import 'vis-timeline/dist/vis-timeline-graph2d.min.css';
+import { constructTimeline } from '../../../helpers/timeline/declarationHelper';
+import { drawTimeline } from '../../../helpers/timeline/drawHelper';
+import { Timeline } from './timeline.style';
 
 const selectors = {
-  TIMELINE: '.timelineContainer .timeline',
+  TIMELINE: '.timeline',
 };
 
+
 const TimelineComponent = ({
   graphJson,
 }) => {
   useEffect(() => {
-    const timelineContainer = document.querySelector(selectors.TIMELINE);
+    const timeline = document.querySelector(selectors.TIMELINE);
```

Comment:
I believe you can add ref on line 39 and use it here.

Reply:
drawTimeline pass it to vis function. Im not sure vis handle react ref

Reply:
You can use `myGreatRefToTheCurrentTimeline.current` - and that should point to the DOM node.